pkg/ebpf/c/vmlinux.h

```diff
@@ -1215,8 +1215,6 @@ struct icmp6hdr {
     } icmp6_dataun;
 };
 
-#include <struct_flavors.h>
```

Comment:
IMPOV it is not a good practice to include headers at the end of other headers (specially if they're so long as our vmlinux).

Either way it would have to be outside the pragma statement (so it has its own pragma or clangd gets confused).

Also, what I said before, contributors usually are confused by types, missing types and flavored types, I wanted them to be distinguishable so dev understand what they're including and why (when creating new C/H files).